packages/vue3/source/src/init.ts

```
@@ -98,23 +90,20 @@ const init = (appConfig, platformConfig, routes, metaData) => {
   if (metaData && metaData.frontendEvents) {
     for (let index = 0; index < endEventLists.length; index++) {
       const name = endEventLists[index];
-      if (name && metaData.frontendEvents[name]) {
+      if (metaData.frontendEvents[name]) {
+        const evalWrap = function (code: string) {
+          code && eval(code);
+        };
         // 确保事件函数中的this指向app
-        evalWrap.bind(app.config.globalProperties)(metaData, name);
+        evalWrap.bind(app.config.globalProperties)(metaData.frontendEvents[name]);
+        app.config.globalProperties[name] = window[name];
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)